staging/src/k8s.io/apiserver/pkg/apis/audit/v1alpha1/types.go

```diff
@@ -220,6 +220,11 @@ type GroupResources struct {
 	// Any empty list implies every resource kind in the API group.
 	// +optional
 	Resources []string `json:"resources,omitempty" protobuf:"bytes,2,rep,name=resources"`
```

Comment:
It does for 1.8, but won't on a 1.7 cluster. I'll expand this comment to mention that uses who want that behavior should just switch to the beta group.